### DISCUSSION TITLE
Scheduling weekday or business day only partition jobs

### DISCUSSION THREAD

**Discussion by kimyun7** [dagster-io/dagster#16274] [Q&A] [answered]
Building from how to example on creating weekday partitions: https://github.com/dagster-io/dagster/discussions/14528

**Let's say if you have a weekday partition like the above example:**
```python
weekday_partition = TimeWindowPartitionsDefinition(cron_schedule="0 0 * * 1-5", start='2021-03-31', fmt='%Y-%m-%d')
```

**With asset job sharing the partition definition:**
```python
weekday_job= define_asset_job(
    name="weekday_job",
    selection=AssetSelection.assets(asset1, asset2),
    partitions_def=weekday_partition,
)
```

**You can build a schedule from the partitioned job but cannot override the hour or minutes**
```python
weekday_schedule = build_schedule_from_partitioned_job(
    weekday_job,
)
```

Same issue with using `get_cron_schedule()`

This is because of the cron syntax doesn't seem to fall under the ScheduleTypes.
If I want to schedule a job at 7am, I could change the hour of day part of the cron syntax in my original partition which does not seem ideal. Any idea how to achieve this?

**Comment by kimyun7** [comment, marked as answer]
Workaround for now:

```python
@schedule(
    cron_schedule="7 00 * * 1-5",
    job=weekday_job,
    execution_timezone="US/Pacific",
    name="weekday_job"
)
def weekday_schedule(context):
    weekday_partitions = weekday_partition .get_partition_keys(context.scheduled_execution_time)
    latest_partition = weekday_partitions[-1]
    yield weekday_job.run_request_for_partition(
        partition_key=current_partition
    )
```

**Comment by sryza** [comment]
Hi @kimyun7 - thanks for posting your workaround.  If there's additional functionality you'd like to request gets added, mind filing a Github issue for it?
